### PR TITLE
Milestone 1 Project 3

### PR DESCRIPTION
**Pull request by iliebabcenco**
This is a clone of Newsweek Page header and footer.
There we have applied Bootstrap.
It has:
- a responsive header;
- a responsive footer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                                style="width: 150px;"></a>
+                    </div>
+                    <div class="col-6 col-lg-4 d-flex justify-content-end align-items-center">
+                        <a href="#" title="user" class="text-white ms-3"><i class="fa fa-user"
+                                aria-hidden="true"></i></a>
+                        <button type="button" class="btn btn-warning rounded fw-bold h6 ms-3 mt-2">SUBSCRIBE
+                        </button>
+                        <a href="#" title="menu" class="text-white ms-3"><i class="fa fa-bars"
+                                aria-hidden="true"></i></a>
+                    </div>
```
Comment (BrittanyBlake) [inline index.html]: - [x] On the desktop screen, the user icon should be the text login and the hamburger icon should not be present. The hamburger and user icons appear on the medium and small screens.
Your clone desktop screen👇🏾 
Screen Shot 2021-03-01 at 8 10 22 PM

Your clone mobile screen 👇🏾 
Screen Shot 2021-03-01 at 8 13 01 PM


The original page desktop screen 👇🏾 
Screen Shot 2021-03-01 at 8 12 22 PM

The original page mobile screen 👇🏾 
Screen Shot 2021-03-01 at 8 13 37 PM

File: index.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                    <li class="col"><a href="#" class="text-decoration-none fw-bold text-dark">Culture</a></li>
+                    <li class="col"><a href="#" class="text-decoration-none fw-bold text-dark">Newsgeen</a></li>
+                    <li class="col"><a href="#" class="text-decoration-none fw-bold text-dark">Sports</a></li>
+                    <li class="col"><a href="#" class="text-decoration-none fw-bold text-dark">Health</a></li>
+                    <li class="col"><a href="#" class="text-decoration-none fw-bold text-dark">The Debate</a></li>
+                    <li class="col"><a href="#" class="text-decoration-none fw-bold text-dark">Experts</a></li>
+                    <li class="col"><a href="#" class="text-decoration-none fw-bold text-dark">Vantage</a></li>
+                    <li class="col"><a href="#" class="text-decoration-none fw-bold text-dark">Search </a><i
+                            class="fa fa-search ms-2" aria-hidden="true"></i>
+                    </li>
```
Comment (BrittanyBlake) [inline index.html]: - [x] This should be a form with a search field.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                </ul>
+
+            </div>
+        </nav>
+    </header>
+    <main></main>
+    <footer style="position: absolute; bottom: 0; left: 0; right: 0;">
+        <div class="container-xl">
+            <div class="row list-unstyled d-flex flex-column flex-sm-row align-items-center">
+                <div class="col mt-2"><img src="./assets/Newsweeklogo2.png" alt=""></div>
```
Comment (BrittanyBlake) [inline index.html]: - [x] The logo in the footer should have a link.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                <div class="col mt-2"> &copy; 2021 NEWSWEEK DIGITAL LLC</div>
+                <div class="col d-flex mt-2 justify-content-sm-end">
+                    <i class="fa fa-facebook-official fa-2x" aria-hidden="true"></i>
+                    <i class="fa fa-twitter-square fa-2x ms-2" aria-hidden="true"></i>
+                    <i class="fa fa-instagram fa-2x ms-2" aria-hidden="true"></i>
+                    <i class="fa fa-tumblr-square fa-2x ms-2" aria-hidden="true"></i>
+                    <i class="fa fa-linkedin-square fa-2x ms-2" aria-hidden="true"></i>
+                </div>
```
Comment (BrittanyBlake) [inline index.html]: - [x] The social media icons should also have links.

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @iliebabcenco and @JulianaOsemeke ! 👋 

You have done an amazing job with this milestone! It looks really good  🤩  You have also done really well ensuring that you are using bootstrap classes everywhere👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****
### GENERAL


- [x] You are missing the dividers between the links in the nav and the footer. Also some of your text is wrapping to another line where it doesn't in the original.

Your clone 👇🏾 
Screen Shot 2021-03-01 at 8 19 20 PM
Screen Shot 2021-03-01 at 8 19 34 PM

The original page 👇🏾 
Screen Shot 2021-03-01 at 8 19 56 PM
Screen Shot 2021-03-01 at 8 20 07 PM


****

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by sergioobolevich — APPROVED** [review]
# STATUS: Approved 🎉🔋👏
Congratulations, awesome job! And thanks for making the required changes.



**You can continue with your next project.**

---

### If you have any question kindly reach me on slack as 'Sergio Obolevich'
Github: [@**chubaquelo**](https://www.github.com/chubaquelo)

### **HAPPY CODING 💻🚀🎉**

**Pull request merged**